### DISCUSSION TITLE
Audio track

### DISCUSSION THREAD

**Discussion by bkil** [asciinema/discussions#90] [Ideas]
It is common for tutorials to include voice guidance either live or added after the recording. Opus sounds reasonable, but some may even go as far as to demand codec2.

I recommend that transcoding (if any) be done on the client side before uploading.

See also asciinema/discussions#89 asciinema/discussions#69 asciinema/discussions#78

**Comment by bkil** [comment]
Related: https://github.com/asciinema/asciinema-server/issues/63

**Comment by bkil** [comment]
See also: https://github.com/asciinema/discussions/issues/6

**Comment by jlb6907** [comment]
+1 for adding voice playback in asciinema-player.
